docs/pbc/all/search/202410.0/base-shop/third-party-integrations/algolia/algolia-personalization.md

```diff
@@ -0,0 +1,355 @@
+                    },
+                    name: {
+                        selector: '.product-name', // Search for an element with the 'product-name' class within the monitored element
+                        attribute: 'price', // Use the 'price' attribute as the value; if absent, fallback to the element's text content
+                    },
+                    price: {
+                        value: 'static value', // Assign a fixed value to the 'price' attribute
+                    },
+                    attributes: {
+                        selector: '.attribute-selector',
+                        multi: true, // Collect all matching elements and return their data as an array
+                    },
+                    metadata: {
+                        multi: true,
+                        selector: '.metadata-row',
+                        composed: { // Create nested structures for more detailed data gathering and start searching elements from `.metadata-row` selector.
+                            brand: {
+                                selector: '.product-brand',
+                                attribute: 'textContent'
+                            },
+                            category: {
+                                selector: '.product-category',
+                            },
+                        },
+                    },
+                },
+            },
+        ],
+    }]),
+    data: events.data,
+} %}
+
+{# Expected transformed data format in the console:
+  {
+      ...global data/event metadata,
+      relatedProducts: {
+          // Flattened data from the 'details' attribute
+          name: VALUE, // The value taken from the 'name' selector or attribute
+          price: 'static value', // The fixed 'price' value
+          attributes: [VALUE, VALUE, VALUE, ...], // Array of values collected from elements matching '.attribute-selector'
+          metadata: [
+              {
+                  brand: VALUE, // 'brand' data extracted from the '.metadata-row .product-brand' element
+                  category: VALUE, // 'category' data from the '.metadata-row .product-category' element
+              },
+              {
+                  brand: VALUE, // 'brand' data extracted from the '.metadata-row .product-brand' element
+                  category: VALUE, // 'category' data from the '.metadata-row .product-category' element
+              },
+              ...
+          ]
+      }
+  }
+#}
+```
+
+##### Examples of the feature integration into Spryker demo shops
+
+- [B2C Demo Shop](https://github.com/spryker-shop/b2c-demo-shop/pull/595/files)
+- [B2C Marketplace Demo Shop](https://github.com/spryker-shop/b2c-demo-marketplace/pull/474/files)
+- [B2B Demo Shop](https://github.com/spryker-shop/b2b-demo-shop/pull/542/files)
+- [B2B Marketplace Demo Shop](https://github.com/spryker-shop/b2b-demo-marketplace/pull/490/files)
+
+  
+
+### Update website agreement text
```

Comment:
Which section is this section related to? Updating Spryker Shop or issues?

Reply:
Updating Spryker Shop

Reply:
there is a difference between level of headers `###` (h3) or `####` (h4)